cura/ApplicationMetadata.py

```diff
@@ -56,3 +56,5 @@
 
 except ImportError:
     CuraAppDisplayName = DEFAULT_CURA_DISPLAY_NAME
+
+WEB_MARKETPLACE_URL = "https://marketplace.ultimaker.com/app/cura/materials"
```

Comment:
This should be dynamic depending on environment (prod vs staging).

Reply:
1. What is the dynamic url? "marketplace-staging...." ?
2. Why would a normal user be interested in the staging environment?

Reply:
1. yup! 
2. Not for end users, but when making internal staging builds for testing it's good to have all the URLs correctly in place to prevent confusion.

Reply:
I've made the URL a build parameter. See also this PR: https://github.com/Ultimaker/cura-build/pull/231

Reply:
Unused, remove